Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -81,6 +83,10 @@ public partial class BlockPolicySource
             new Address("636d187B4d434244A92B65B06B5e7da14b3810A9"),
         }.ToImmutableHashSet();
 
+        public static readonly Address PreviewNetAdmin = new Address(
```

Comment:
Where is this field used? If it is to check the address only in the action, AdminState already exists.